cli/local_start.sh

```diff
@@ -39,6 +39,7 @@ AWS_REGION=test PORT=8084 DB_URL=postgres://gradejs:gradejs@localhost:5432/grade
   SQS_LOCAL_PORT=29324 AWS_ACCESS_KEY_ID=secret AWS_SECRET_ACCESS_KEY=secret \
   INTERNAL_API_ROOT_URL=http://localhost:8082 \
   GRADEJS_API_KEY=TEST_API_KEY \
+  ROLLBAR_API_KEY=test \
```

Comment:
I'd still heavily insist on making this env var optional @spalt08

Reply:
Ok, done